src/components/SettlementsList.js

```diff
@@ -59,7 +58,7 @@ function SettlementsList(props) {
     getSettlements(fsp, dates, { ftc })
       .then(setSettlements)
       .catch(ftc.ignoreAbort())
-      .catch(err => window.alert('Failed to get settlement list')); // TODO: better error message, let user retry
+      .catch(() => window.alert('Failed to get settlement list')); // TODO: better error message, let user retry
```

Comment:
As we're here, could we console log this error instead?
```suggestion
      .catch((err) => {
        window.alert('Failed to get settlement list'); // TODO: better error message, let user retry
        console.log('Failed to get settlement list', err);
      });
```